### PR TITLE
Development

### PR DESCRIPTION
**Pull request by YemreAybey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
...
+**Structure of Smashing Magazine's Main Page**      
+This project was created by:
+1. [Onur Onat SURMELI](https://github.com/Zibilyonik)
+2. [Yunus Emre AYBEY](https://github.com/YemreAybey)  
+using:
+- *HTML, HTML5*
+- *CSS3*   
+You can find the live version [here](https://raw.githack.com/YemreAybey/TearDown/development/index.html)
+
+**Have a Nice Day**
```
Comment (mcrd25) [inline README.md:11]: Thank you 😸! I will!

File: assets/CSS/reset.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
```
Comment (mcrd25) [inline assets/CSS/reset.css:48]: Good job having a resect.css file 👍

File: README.md (hunk 1 above)
Comment (mcrd25) [reply README.md:11]: Also, great that you have a descriptive README without being prompted. 🥇

File: assets/CSS/style.css [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+  font-weight: bold;
+}
+.latest-articles-title {
+  font-size: 1.5em;
+  font-weight: bold;
+  display: block;
+  position: absolute;
+  top: 150px;
+  left: 45%;
+  text-align: center;
```
Comment (mcrd25) [inline assets/CSS/style.css]: This line appears 10 times in the stylesheet, try using a separate class selector for repeating classes like these.

Comment (mcrd25) [reply assets/CSS/style.css]: `display: grid` is also a good candidate for same. Of course, you are free to disagree with me on this, simply comment here :)

**Review by mcrd25 — COMMENTED** [review]
# Wow! Really great project! 🥇 💯 👍 . You can 🚀 

## But wait

I know you guys are always willing to try new things. Honestly this project is great but there is one thing that bothers me.

- *Don't Miss These Articles* should be centered.
 
Also, maybe try the class for duplicate styling if you so, please.

If you choose not to do this, that's fine you can submit. 
However, if you do that'd be awesome! Let me know and submit another code review or you can simply message me here and implicitly approve.

Good attempt. No other comments since this project are more based on what you guys think is most important. You have an idea of how heat maps work.

However, was wondering if you guys thought the *Don’t Miss These Latest
Articles* is something that you take notice of immediately. Maybe this should be black or darker rather than white? Is it the text that attracts or the white background (contrasting with the red) that first attracts your attention. 

Look forward to your feedback. Great project again 👍 💯 🥇  😄

File: assets/CSS/style.css (hunk 3 above)
Comment (YemreAybey) [reply assets/CSS/style.css]: The "text-align" property has been put in a common class (class="center")
display: grid is also better for us we changed it :+1: :+1:  
Don't Miss... part has been centered (We hope so :D)
some classes which have the same properties (more than one) have been combined together in CSS

And also thank you for your great review Maya :1st_place_medal: 
We are gonna send another review from the dashboard hope to see you :wave:

Comment (mcrd25) [reply assets/CSS/style.css]: I am glad you liked my review! 😄 . I Like Y'all project. Checking out changes now.

**Review by mcrd25 — APPROVED** [review]
# Great Job! You may merge and move on! 🚀 

Looking forward to hopefully reviewing more of Y'all projects. You guys are willing to improve your project which is always great.

👋  Bye for now. Keep up the good work 👍 💯 😄 . The project is well done!

**Pull request merged**